hackathon_site/dashboard/frontend/src/slices/order/orderSlice.ts

```diff
@@ -89,6 +112,29 @@ const orderSlice = createSlice({
                 payload?.message ||
                 "There was a problem retrieving orders. If this continues please contact hackathon organizers.";
         });
+
+        builder.addCase(updateOrder.pending, (state) => {
+            state.isLoading = true;
+            state.error = null;
+        });
+
+        builder.addCase(updateOrder.fulfilled, (state, { payload }) => {
+            state.isLoading = false;
+            state.error = null;
+            if (payload) {
+                pendingOrderAdapter.updateOne(state, {
+                    id: payload.id,
+                    changes: payload,
+                });
+            }
+        });
+
+        builder.addCase(updateOrder.rejected, (state, { payload }) => {
+            state.isLoading = false;
+            state.error =
```

Comment:
Actually, I don't think we need an error state, you can just dispatch a snackbar here with the error message

Reply:
I can dispatch a snack bar where I catch errors in the thunk, I don't know if I can dispatch in this rejected case.